SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -92,7 +92,10 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 	LocalDeploymentManager = GDKServices.GetLocalDeploymentManager();
 	LocalDeploymentManager->PreInit(GetDefault<USpatialGDKSettings>()->IsRunningInChina());
 
+	LocalReceptionistProxyServerManager = GDKServices.GetLocalReceptionistProxyServerManager();
+
 	OnAutoStartLocalDeploymentChanged();
+	OnAutoStartLocalReceptionistProxyServer();
```

Comment:
Just wanted to understand why when we start the module we need to attempt to start the proxy?

Reply:
In case the option to startlocalserver is already enabled when you start the editor. We have to make sure that we bind the delegate as the option is enabled and the proxy will run when you click play only is it is bound.